src/python/k4a/examples/point_cloud_capture.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import k4a
+
+if __name__ == '__main__':
+    # Open Device
+    device = k4a.Device.open()
+
+    # Start Cameras
+    device_config = k4a.DEVICE_CONFIG_BGRA32_1080P_NFOV_UNBINNED_FPS15
+    device.start_cameras(device_config)
+
+    # Get Calibration
+    calibration = device.get_calibration(
+        depth_mode=device_config.depth_mode,
+        color_resolution=device_config.color_resolution)
+
+    # Create Transformation
+    transformation = k4a.Transformation(calibration)
+
+    # Capture One Frame
+    capture = device.get_capture(-1)
+
+    # Get Point Cloud
+    xyz_image = transformation.depth_image_to_point_cloud(capture.depth, k4a.ECalibrationType.DEPTH)
+
+    # Save Point Cloud To Ascii Format File. Interleave the [X, Y, Z] channels into [x0, y0, z0, x1, y1, z1, ...]
+    width, height, channels = xyz_image.data.shape
+    xyz_data = np.empty((width * height, channels,), dtype=xyz_image.data.dtype)
```

Comment:
I think this line is not required. Please consider to remove this line.

Reply:
Removed with most recent update.